docs/en/02_Developer_Guides/09_Security/06_Personal_Data.md

```diff
@@ -1,32 +1,32 @@
 ---
 title: Personal Data
-summary: How the SilverStripe CMS deals with data privacy
+summary: How the Silverstripe CMS deals with data privacy
 icon: user-ninja
 ---
 
 # Personal Data
 
-SilverStripe is an application framework which can be used to process
+Silverstripe CMS is an application framework which can be used to process
 and store data. Any data can be sensitive, particularly if it is
 considered personal data. Many regulatory frameworks such as the
 [EU General Data Protection Regulation (GDPR)](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation)
 can be interpreted to regard basic data points such as email and IP addresses
 as personally identifiable data.
 
 This document is aiding implementors and auditors in determining
-the impact of using the SilverStripe Framework and CMS
+the impact of using the Silverstripe CMS Framework and CMS
```

Comment:
please change to `[...] using the Silverstripe CMS to build online services. [...]` (remove `Framework and CMS`)

Reply:
Framework and CMS removed